### PR TITLE
[develop] Add Gaea C5 to supported platforms

### LINKED ISSUE #886: Add Gaea C5 to supported platforms, as Tier-1 system
Add support for Gaea C5 system at GFDL as Tier-1 platform. Include standard SRW testing suites. 

## Solution
- Prepare modulefiles
- prepare configuration files for standard testing suites
- prepare modules and environments for running the workflow
- prepare machine files 

## Requirements**
- Use updated software stacks currently used by UFS-Weather-Model (hdf5/1.14.0, netcdf/4.9.2-based)
-  rocoto build for Hercules
-  workflow python3/conda environments for the SRW tasks 
- no need for additional data staging, as Gaea C5 shares the filesystem with Gaea C4/C3 at GFDL

## Acceptance Criteria (Definition of Done)

All standard tests successfully run and pass on Gaea C5 using manual and automated Jenkins testings

## Dependencies 

Depends on https://github.com/ufs-community/ufs-weather-model/issues/1784

### PR DESCRIPTION
**Pull request by natalie-perlin**
Modulefiles and other configuration files that complete porting the SRW to Gaea C5 system.

Software stacks used for testing are hdf5/1.14.0, netcdf/4.9.2-based, similar to those used in https://github.com/ufs-community/ufs-srweather-app/pull/889.

## DESCRIPTION OF CHANGES: 
Add Gaea C5 at GFDL as NOAA RDHPCS supported system

### Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [x] This change requires a documentation update

## TESTS CONDUCTED: 
All fundamental tests pass successfully on Gaea_c5
All comprehensive tests pass except one (nco_grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_HRRR) that fails on some other platforms as well.

- [ ] hera.intel
- [ ] orion.intel
- [x] gaea_c5.intel
- [ ] gaea.intel
- [ ] jet.intel
- [ ] wcoss2.intel
- [ ] NOAA Cloud (indicate which platform)
- [ ] Jenkins

## DEPENDENCIES:
Depends on https://github.com/ufs-community/ufs-srweather-app/pull/889 - MERGED

## ISSUE: 
Fixes issue https://github.com/ufs-community/ufs-srweather-app/issues/886

## CHECKLIST

- [x] My code follows the style guidelines in the Contributor's Guide
- [x] I have performed a self-review of my own code using the Code Reviewer's Guide
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] My changes need updates to the documentation. I have made corresponding changes to the documentation
- [ ] My changes do not require updates to the documentation (explain).
- [x] My changes generate no new warnings
- [ ] New and existing tests pass with my changes
- [x] Any dependent changes have been merged and published

## LABELS (optional): 

- [ ] Work In Progress
- [ ] bug
- [x] enhancement
- [ ] documentation
- [x] release
- [x] high priority
- [ ] run_ci
- [x] run_we2e_fundamental_tests
- [ ] run_we2e_comprehensive_tests
- [ ] Needs Jet test 
- [ ] Needs Hera test 
- [ ] Needs Orion test 
- [ ] help wanted

## CONTRIBUTORS (optional): 
@RatkoVasic-NOAA - thank you for you contribution!!

[WE2E_gaea_c5_fundamental_summary.txt](https://github.com/ufs-community/ufs-srweather-app/files/12583676/WE2E_gaea_c5_fundamental_summary.txt)

### CODE REVIEW

File: etc/lmod-setup.csh [hunk 1]
```diff
@@ -40,6 +40,9 @@ else if ( "$L_MACHINE" == singularity ) then
 else if ( "$L_MACHINE" == gaea ) then
    source /lustre/f2/dev/role.epic/contrib/Lmod_init.csh
 
+else if ( "$L_MACHINE" == gaea_c5 ) then
+   source /lustre/f2/dev/role.epic/contrib/Lmod_init_C5.csh
```
Comment (MichaelLueken) [inline etc/lmod-setup.csh:44]: There is no file named `Lmod_init_C5.csh` in `/lustre/f2/dev/role.epic/contrib`:
```
gaea58 Michael.Lueken/ufs-srweather-app> ls /lustre/f2/dev/role.epic/contrib/Lmod_init_C5.csh
ls: cannot access '/lustre/f2/dev/role.epic/contrib/Lmod_init_C5.csh': No such file or directory
```

**Review by MichaelLueken — COMMENTED** [review]
@natalie-perlin - I was able to successfully clone your fork, checkout the develop_gaea_c5 branch, build the SRW App, and submit the fundamental tests.  However, I should note that while attempting to use `source etc/lmod-setup.csh gaea_c5`, it failed due to no file named `Lmod_init_C5.csh` being in `/lustre/f2/dev/role.epic/contrib`.  I also found it interesting that I had to load python before I could use `./manage_externals/checkout_externals`.

Similar to what you are encountering, the tests are failing in the `make_sfc_climo` task.  I haven't seen this particular NetCDF error before, but looking up the error message:
```
FATAL ERROR: ERROR IN NF90_CREATE: Permission denied
 STOP.
```
it looks like we don't have permission to create the necessary NetCDF file.  It isn't clear to me why this would be the case, unless `make_sfc_climo` is attempting to create a file in the EPIC role account space, rather than in my own local directory.

Since `sfc_climo_gen.fd` is in UFS_UTILS, is it possible that we need to do something similar there as we had to do with sorc/ufs-weather-model/cmake?

**Comment by natalie-perlin** [comment]
@MichaelLueken - fixed the file Lmod_init_C5.csh

**Comment by natalie-perlin** [comment]
Fundamental tests pass successfully on Gaea_c5:
```
All 7 experiments finished
Calculating core-hour usage and printing final summary
----------------------------------------------------------------------------------------------------
Experiment name                                                  | Status    | Core hours used 
----------------------------------------------------------------------------------------------------
grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta    COMPLETE              18.53
nco_grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_timeoffset_suite_  COMPLETE              25.05
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2        COMPLETE              13.45
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v17_p8_plot  COMPLETE              28.05
grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_HRRR_suite_HRRR          COMPLETE              32.42
grid_SUBCONUS_Ind_3km_ics_HRRR_lbcs_RAP_suite_WoFS_v0              COMPLETE              32.68
grid_RRFS_CONUS_25km_ics_NAM_lbcs_NAM_suite_GFS_v16                COMPLETE              46.00
----------------------------------------------------------------------------------------------------
Total                                                              COMPLETE             196.18

Detailed summary written to /lustre/f2/scratch/ncep/Natalie.Perlin/C5/SRW/expt_dirs/WE2E_summary_20230912032617.txt
```
Thank you to @RatkoVasic-NOAA for debugging help with wrapper scripts!

[WE2E_gaea_c5_fundamental_summary.txt](https://github.com/ufs-community/ufs-srweather-app/files/12583731/WE2E_gaea_c5_fundamental_summary.txt)

**Comment by natalie-perlin** [comment]
Tested the option of running the SRW on Gaea C5 without addition cmake options files for the UFS WM. Gaea C5 fundamental tests all pass, except for the nco test, which had troubles in get_ics, get_lbcs, and not related to other tasks. 
With the WM avoiding “preliminary changes for Derecho, Gaea C5, Hercules”, I attempted to test the ported SRW on these machines without specific changes for the cmake options in UFS WM. It worked OK for Gaea C5 for all fundamental tests, except for the nco test, which had troubles in get_ics, get_lbcs, and not related to other tasks. I’m working to solve it, to reduce chances of timeouts by increasing wait time and time allowed for the task. We could proceed with testing. 

```
All 7 experiments finished
Calculating core-hour usage and printing final summary
----------------------------------------------------------------------------------------------------
Experiment name                                                  | Status    | Core hours used 
----------------------------------------------------------------------------------------------------
grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta    COMPLETE              18.70
nco_grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_timeoffset_suite_  DEAD                   0.02
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2        COMPLETE              12.97
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v17_p8_plot  COMPLETE              27.60
grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_HRRR_suite_HRRR          COMPLETE              31.97
grid_SUBCONUS_Ind_3km_ics_HRRR_lbcs_RAP_suite_WoFS_v0              COMPLETE              32.82
grid_RRFS_CONUS_25km_ics_NAM_lbcs_NAM_suite_GFS_v16                COMPLETE              46.12
----------------------------------------------------------------------------------------------------
Total                                                              DEAD                 170.20

Detailed summary written to /lustre/f2/scratch/ncep/Natalie.Perlin/C5/SRW/expt_dirs/WE2E_summary_20230914234457.txt
```

Status of the _grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_timeoffset_suite_GFS_v16 test:
```
(workflow_tools) [Natalie.Perlin@gaea54:/lustre/f2/scratch/ncep/Natalie.Perlin/C5/SRW/expt_dirs/nco_grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_timeoffset_suite_GFS_v16]$ rocotostat -w FV3LAM_wflow.xml -d FV3LAM_wflow.db -v 10
       CYCLE                    TASK                       JOBID               STATE         EXIT STATUS     TRIES      DURATION
================================================================================================================================
202208101200           get_extrn_ics                    76264086                DEAD                   1         1          21.0
202208101200          get_extrn_lbcs                    76264087                DEAD                   1         1          24.0
202208101200         make_ics_mem000                           -                   -                   -         -             -
202208101200        make_lbcs_mem000                           -                   -                   -         -             -
```

Tasks get_ics and get_lbcs finish successfully after rerunning them with longer wait time for retrieval process:
```
(workflow_tools) [Natalie.Perlin@gaea54:/lustre/f2/scratch/ncep/Natalie.Perlin/C5/SRW/expt_dirs/nco_grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_timeoffset_suite_GFS_v16]$ rocotostat -w FV3LAM_wflow.xml -d FV3LAM_wflow.db -v 10
       CYCLE                    TASK                       JOBID               STATE         EXIT STATUS     TRIES      DURATION
================================================================================================================================
202208101200           get_extrn_ics                    76265154           SUCCEEDED                   0         1         455.0
202208101200          get_extrn_lbcs                    76265155           SUCCEEDED                   0         1        1460.0
202208101200         make_ics_mem000                   134570288           SUCCEEDED                   0         1         105.0
202208101200        make_lbcs_mem000                   134570289           SUCCEEDED                   0         1         165.0
202208101200         run_fcst_mem000    druby://gaea54.ncrc.gov:44859          SUBMITTING                   -         0           0.0
```

**Comment by natalie-perlin** [comment]
A remaining nco fundamental test successful after adjusting time for the get_lbcs task (it occasionally failed due to time limit):
```
Calculating core-hour usage and printing final summary
----------------------------------------------------------------------------------------------------
Experiment name                                                  | Status    | Core hours used 
----------------------------------------------------------------------------------------------------
nco_grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_timeoffset_suite_  COMPLETE              25.73
----------------------------------------------------------------------------------------------------
Total                                                              COMPLETE              25.73

Detailed summary written to /lustre/f2/scratch/ncep/Natalie.Perlin/C5/SRW/expt_dirs/C5/WE2E_summary_20230915103031.txt
```

**Comment by natalie-perlin** [comment]
All comprehensive tests except one pass for Gaea C5:

```
All 62 experiments finished
Calculating core-hour usage and printing final summary
----------------------------------------------------------------------------------------------------
Experiment name                                                  | Status    | Core hours used 
----------------------------------------------------------------------------------------------------
2020_CAD                                                           COMPLETE              35.53
community                                                          COMPLETE              43.42
custom_ESGgrid                                                     COMPLETE              13.49
custom_ESGgrid_Central_Asia_3km                                    COMPLETE              32.01
custom_ESGgrid_IndianOcean_6km                                     COMPLETE              22.94
custom_ESGgrid_NewZealand_3km                                      COMPLETE              48.87
custom_ESGgrid_Peru_12km                                           COMPLETE              23.10
custom_ESGgrid_SF_1p1km                                            COMPLETE             161.18
custom_GFDLgrid__GFDLgrid_USE_NUM_CELLS_IN_FILENAMES_eq_FALSE      COMPLETE               8.63
custom_GFDLgrid                                                    COMPLETE               8.12
deactivate_tasks                                                   COMPLETE               0.89
get_from_AWS_ics_GEFS_lbcs_GEFS_fmt_grib2_2022040400_ensemble_2me  COMPLETE             693.09
get_from_NOMADS_ics_FV3GFS_lbcs_FV3GFS                             COMPLETE              14.46
grid_CONUS_25km_GFDLgrid_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16      COMPLETE              15.75
grid_CONUS_3km_GFDLgrid_ics_FV3GFS_lbcs_FV3GFS_suite_RRFS_v1beta   COMPLETE             237.93
grid_RRFS_AK_13km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16_plot        COMPLETE             129.98
grid_RRFS_AK_3km_ics_FV3GFS_lbcs_FV3GFS_suite_HRRR                 COMPLETE             171.46
grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_RAP              COMPLETE              30.46
grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16_plot     COMPLETE              35.18
grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_HRRR             COMPLETE              32.58
grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_RRFS_v1beta      COMPLETE              30.84
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2        COMPLETE              12.14
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16_plot     COMPLETE              26.37
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v17_p8_plot  COMPLETE              26.06
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_HRRR             COMPLETE              38.73
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_RAP              COMPLETE              71.05
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_RRFS_v1beta      COMPLETE              40.01
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_RAP_suite_RAP                 COMPLETE              19.07
grid_RRFS_CONUS_25km_ics_GSMGFS_lbcs_GSMGFS_suite_GFS_v15p2        COMPLETE              12.82
grid_RRFS_CONUS_25km_ics_NAM_lbcs_NAM_suite_GFS_v16                COMPLETE              45.89
grid_RRFS_CONUS_25km_ics_NAM_lbcs_NAM_suite_RRFS_v1beta            COMPLETE              36.85
grid_RRFS_CONUS_3km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2         COMPLETE             235.82
grid_RRFS_CONUS_3km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15_thompson  COMPLETE             315.97
grid_RRFS_CONUS_3km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16           COMPLETE             326.71
grid_RRFS_CONUS_3km_ics_FV3GFS_lbcs_FV3GFS_suite_HRRR              COMPLETE             347.92
grid_RRFS_CONUS_3km_ics_FV3GFS_lbcs_FV3GFS_suite_RRFS_v1beta       COMPLETE             347.71
grid_RRFS_CONUScompact_13km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16   COMPLETE              29.92
grid_RRFS_CONUScompact_13km_ics_HRRR_lbcs_RAP_suite_HRRR           COMPLETE              28.58
grid_RRFS_CONUScompact_13km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta    COMPLETE              27.04
grid_RRFS_CONUScompact_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16   COMPLETE              18.38
grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_HRRR_suite_HRRR          COMPLETE              31.30
grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta    COMPLETE              17.97
grid_RRFS_CONUScompact_3km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16    COMPLETE             262.52
grid_RRFS_CONUScompact_3km_ics_HRRR_lbcs_RAP_suite_HRRR            COMPLETE             277.47
grid_RRFS_CONUScompact_3km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta     COMPLETE             276.21
grid_RRFS_NA_13km_ics_FV3GFS_lbcs_FV3GFS_suite_RAP                 COMPLETE              77.54
grid_SUBCONUS_Ind_3km_ics_FV3GFS_lbcs_FV3GFS_suite_WoFS_v0         COMPLETE              33.11
grid_SUBCONUS_Ind_3km_ics_HRRR_lbcs_HRRR_suite_HRRR                COMPLETE              40.27
grid_SUBCONUS_Ind_3km_ics_HRRR_lbcs_RAP_suite_WoFS_v0              COMPLETE              32.54
grid_SUBCONUS_Ind_3km_ics_NAM_lbcs_NAM_suite_GFS_v16               COMPLETE              52.25
grid_SUBCONUS_Ind_3km_ics_RAP_lbcs_RAP_suite_RRFS_v1beta_plot      COMPLETE              16.91
MET_ensemble_verification_only_vx                                  COMPLETE               1.05
MET_verification_only_vx                                           COMPLETE               0.19
nco                                                                COMPLETE              21.62
nco_ensemble                                                       COMPLETE             104.55
nco_grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16      COMPLETE              31.21
nco_grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_timeoffset_suite_  COMPLETE              25.47
nco_grid_RRFS_CONUS_3km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15_thom  COMPLETE             309.08
nco_grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_HRRR       DEAD                   1.19
pregen_grid_orog_sfc_climo                                         COMPLETE              12.96
specify_EXTRN_MDL_SYSBASEDIR_ICS_LBCS                              COMPLETE              11.71
specify_template_filenames                                         COMPLETE              13.49
----------------------------------------------------------------------------------------------------
```

The test nco_grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_HRRR does not pass on other machines tested - Gaea, Orion, Derecho.

**Review by MichaelLueken — APPROVED** [review]
@natalie-perlin - Thanks for your work on porting the SRW App to Gaea C5!  Your branch was cloned and built using the Jenkins build script (`.cicd/scripts/srw_build.sh`) and the `coverage.gaea_c5` test suite was run using the Jenkins test script (`.cicd/scripts/srw_test.sh`).  All coverage tests successfully passed:
```
----------------------------------------------------------------------------------------------------
Experiment name                                                  | Status    | Core hours used 
----------------------------------------------------------------------------------------------------
community                                                          COMPLETE              43.50
custom_ESGgrid_NewZealand_3km                                      COMPLETE              48.17
grid_RRFS_CONUScompact_13km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta    COMPLETE              26.46
grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_RAP              COMPLETE              30.58
grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_HRRR             COMPLETE              30.32
grid_RRFS_CONUS_3km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15_thompson  COMPLETE             312.60
grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_HRRR_suite_HRRR          COMPLETE              30.80
grid_RRFS_CONUScompact_3km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta     COMPLETE             274.60
grid_SUBCONUS_Ind_3km_ics_RAP_lbcs_RAP_suite_RRFS_v1beta_plot      COMPLETE              17.23
nco_ensemble                                                       COMPLETE              99.47
nco_grid_RRFS_CONUS_3km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15_thom  COMPLETE             307.34
----------------------------------------------------------------------------------------------------
Total                                                              COMPLETE            1221.07
```
Approving this PR now.

**Review by RatkoVasic-NOAA — APPROVED** [review]
Tested on C5.

**Comment by MichaelLueken** [comment]
@natalie-perlin - While submitting the Jenkins tests, I noted that the `gaea_c5` runner wasn't being submitted.  Looking at the pipeline.log file, I saw that there is no `gaea_c5` label.  I reached out to Kris and the platform team on Slack and Kris noted that the Gaea C5 label should be `gaea-c5`.  If you could go through and replace the underscore with a hyphen, I would greatly appreciate it!  Please let me know if you need assistance, and I will attempt to make these changes to your branch.  Thank you very much!

**Comment by MichaelLueken** [comment]
The WE2E coverage tests were successfully run on Derecho:
```
----------------------------------------------------------------------------------------------------
Experiment name                                                  | Status    | Core hours used 
----------------------------------------------------------------------------------------------------
custom_ESGgrid_IndianOcean_6km                                     COMPLETE              21.93
grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16_plot     COMPLETE              35.76
grid_RRFS_CONUS_25km_ics_NAM_lbcs_NAM_suite_GFS_v16                COMPLETE              42.79
grid_RRFS_CONUScompact_13km_ics_HRRR_lbcs_RAP_suite_HRRR           COMPLETE              26.37
grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta    COMPLETE              16.38
grid_SUBCONUS_Ind_3km_ics_HRRR_lbcs_HRRR_suite_HRRR                COMPLETE              38.75
nco_grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_timeoffset_suite_  COMPLETE              22.16
pregen_grid_orog_sfc_climo                                         COMPLETE              13.49
specify_template_filenames                                         COMPLETE              13.97
----------------------------------------------------------------------------------------------------
Total                                                              COMPLETE             231.60
```

**Comment by natalie-perlin** [comment]
@MichaelLueken @RatkoVasic-NOAA - all instances of "gaea_c5" renamed to "gaea-c5" ( a file or directory name, or a string in a file )

**Comment by MichaelLueken** [comment]
@natalie-perlin - Unfortunately, following the merge of PR #911, there is now a conflict with this PR in `.cicd/Jenkinsfile`, `tests/build.sh`, and `ush/valid_param_vals.yaml`.  Please merge this morning's hash update ([87dbf19](https://github.com/ufs-community/ufs-srweather-app/commit/87dbf19f9d6ae9f9355a4229bfd25302d7a0b9a4)) to your `develop_gaea_c5` branch, address the three conflicts, then I should be able to requeue the Jenkins tests for this PR.

**Comment by MichaelLueken** [comment]
@natalie-perlin - Thank you for merging the latest HEAD into your branch and correcting the conflicts!  I have requeued the Jenkins tests for this PR and will let you know if there are any issues.

**Comment by MichaelLueken** [comment]
The Jenkins Hera Intel tests have completed:
```
----------------------------------------------------------------------------------------------------
Experiment name                                                  | Status    | Core hours used
----------------------------------------------------------------------------------------------------
custom_ESGgrid_Central_Asia_3km                                    DEAD                   6.63
get_from_HPSS_ics_FV3GFS_lbcs_FV3GFS_fmt_grib2_2019061200          COMPLETE               6.30
get_from_HPSS_ics_GDAS_lbcs_GDAS_fmt_netcdf_2022040400_ensemble_2  COMPLETE             757.90
get_from_HPSS_ics_HRRR_lbcs_RAP                                    COMPLETE              14.01
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2        COMPLETE               6.30
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16_plot     COMPLETE              13.01
grid_RRFS_CONUS_25km_ics_FV3GFS_lbcs_RAP_suite_RAP                 COMPLETE              10.13
grid_RRFS_CONUS_25km_ics_GSMGFS_lbcs_GSMGFS_suite_GFS_v15p2        COMPLETE               6.65
grid_RRFS_CONUS_3km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15p2         COMPLETE             231.40
grid_RRFS_CONUS_3km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v16           COMPLETE             303.34
grid_RRFS_CONUScompact_3km_ics_HRRR_lbcs_RAP_suite_HRRR            COMPLETE             324.52
pregen_grid_orog_sfc_climo                                         COMPLETE               8.07
----------------------------------------------------------------------------------------------------
Total                                                              DEAD                1688.26
```
A rerun on the `custom_ESGgrid_Central_Asia_3km` test shows successful completion:
```
----------------------------------------------------------------------------------------------------
Experiment name                                                  | Status    | Core hours used 
----------------------------------------------------------------------------------------------------
custom_ESGgrid_Central_Asia_3km                                    COMPLETE              24.99
----------------------------------------------------------------------------------------------------
Total                                                              COMPLETE              24.99
```
The Gaea and Hercules tests have successfully completed.  The Jet tests are still running.  The Gaea C5 and Orion tests have been requeued.  Once the tests complete, I will move forward with merging this PR.

**Comment by MichaelLueken** [comment]
@natalie-perlin - The Jenkins tests successfully passed on Hera GNU, Jet, Orion.  The Gaea C5 `Functional Workflow Task Tests` failed with the following message:
```
Running: sbatch -A epic --parsable /lustre/f2/dev/role.epic/jenkins/workspace/fs-srweather-app_pipeline_PR-898/.cicd/scripts/sbatch_srw_ftest.sh

sbatch: error: Batch job submission failed: Invalid qos specification
```
It looks like you will also need to add the:

`sed -i 's|qos=batch|qos=windfall|g' ${WORKSPACE}/.cicd/scripts/${workflow_cmd}_srw_ftest.sh`

to the Gaea C5 section of `.cicd/scripts/wrapper_srw_ftest.sh`, replacing `qos=windfall` with `qos=normal`.  With this change, I'll kick off the Jenkins tests for Gaea C5 again.

**Comment by natalie-perlin** [comment]
@MichaelLueken - done with changes for Gaea C5 in `.cicd/scripts/wrapper_srw_ftest.sh`, adding a line `sed -i 's|qos=batch|qos=normal|g' ${WORKSPACE}/.cicd/scripts/${workflow_cmd}_srw_ftest.sh`

**Comment by MichaelLueken** [comment]
@natalie-perlin - Thanks!  Requeuing the Jenkins tests for Gaea C5 now.  I'll let you know if any other issues arise.

**Comment by MichaelLueken** [comment]
The latest Jenkins tests successfully passed on Gaea C5:
```
----------------------------------------------------------------------------------------------------
Experiment name                                                  | Status    | Core hours used 
----------------------------------------------------------------------------------------------------
community                                                          COMPLETE              43.58
custom_ESGgrid_NewZealand_3km                                      COMPLETE              49.38
grid_RRFS_CONUScompact_13km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta    COMPLETE              27.39
grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_RAP              COMPLETE              30.87
grid_RRFS_CONUS_13km_ics_FV3GFS_lbcs_FV3GFS_suite_HRRR             COMPLETE              31.52
grid_RRFS_CONUS_3km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15_thompson  COMPLETE             318.02
grid_RRFS_CONUScompact_25km_ics_HRRR_lbcs_HRRR_suite_HRRR          COMPLETE              31.69
grid_RRFS_CONUScompact_3km_ics_HRRR_lbcs_RAP_suite_RRFS_v1beta     COMPLETE             277.13
grid_SUBCONUS_Ind_3km_ics_RAP_lbcs_RAP_suite_RRFS_v1beta_plot      COMPLETE              17.73
nco_ensemble                                                       COMPLETE             102.79
nco_grid_RRFS_CONUS_3km_ics_FV3GFS_lbcs_FV3GFS_suite_GFS_v15_thom  COMPLETE             312.02
----------------------------------------------------------------------------------------------------
Total                                                              COMPLETE            1242.12
```
Since Orion and Hercules are down, manual submission of the Jenkins pipeline caused jobs to be kicked off on these machines, which were aborted.

Moving forward with merging this PR now.

**Comment by BruceKropp-Raytheon** [comment]
I can confirm this PR code can build SRW on gaea-c5, and produce a E2E plot.

**Pull request merged**